### PR TITLE
ecs: add v8 preview using v1 implementation

### PR DESCRIPTION
**Pull request by yaauie**
Thanks for contributing to Logstash! If you haven't already signed our CLA, here's a handy link: https://www.elastic.co/contributor-agreement/

### CODE REVIEW

File: spec/filters/grok_spec.rb [hunk 1]
```diff
@@ -701,7 +703,8 @@ def self.sample(message, &block)
       expect( LogStash::Json.dump(event.get('username')) ).to eql "\"testuser\""
 
       expect( event.to_json ).to match %r|"src_ip":"1.1.1.1"|
-      expect( event.to_json ).to match %r|"@timestamp":"20\d\d-\d\d-\d\dT\d\d:\d\d:\d\d\.\d\d\dZ"|
+      # expect( event.to_json ).to match %r|"@timestamp":"20\d\d-\d\d-\d\dT\d\d:\d\d:\d\d(\.(\d\d\d){1,3})?Z"|
```
Comment (robbavey) [inline spec/filters/grok_spec.rb:706]: Did you intend to leave this commented out?

Comment (yaauie) [reply spec/filters/grok_spec.rb:706]: I intended to delete it. The next line replicates what it was attempting to validate.

Comment (yaauie) [reply spec/filters/grok_spec.rb:706]:
```suggestion
```

**Review by robbavey — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+## 4.4.1
+ - Added preview of ECS v8 support using existing ECS v1 implementation
```
Comment (yaauie) [inline CHANGELOG.md:2]:
```suggestion
 - Added preview of ECS v8 support using existing ECS v1 implementation [#175](https://github.com/logstash-plugins/logstash-filter-grok/pull/175)
```

**Pull request merged**